dtmi/com/hms_networks/ewon_flexy_azure_connector_example-1.json

```diff
@@ -0,0 +1,24 @@
+{
+	"@context": "dtmi:dtdl:context;2",
+	"@id": "dtmi:com:hms_networks:Ewon_Flexy_Azure_Connector_Example;1",
+	"@type": "Interface",
+	"displayName": "Ewon_Flexy_Azure_Connector_Example",
+	"contents": [
+		{
+			"@type": "Property",
+			"name": "DeviceMode",
+			"writable": true,
+			"schema": "string"
+		},
+		{
+			"@type": "Telemetry",
+			"name": "ConnectorControl",
+			"schema": "boolean"
+		},
+		{
+			"@type": "Telemetry",
+			"name": "EXAMPLE",
```

Comment:
What means `EXAMPLE` ? In this repo we'd like to keep *production ready* interfaces, and not examples.

Reply:
Hi, I have removed this example interface. This PR now includes only _production ready_ interfaces.